src/js/ui/menu.js

```diff
@@ -304,6 +310,11 @@ Menu.prototype.selection = function(callback) {
   simply.impl.menuSelection();
 };
 
+Menu.prototype.setSelection = function(sectionIndex, itemIndex) {
+  this._selection = { sectionIndex: sectionIndex, itemIndex: itemIndex };
+  this._select();
+};
+
```

Comment:
Sorry, but what does `selection` do? It's not documented everywhere, and I can't make it out from the source code.

I think it'll make more sense to rename the method to `selected` and make it return the currently selected indices when called without parameters.